pkg/model/participation/question.go

```diff
@@ -21,93 +19,78 @@ var (
 	answersArrayRules = &serializer.ArrayRules{
 		Min:            QuestionMinAnswersCount,
 		Max:            QuestionMaxAnswersCount,
-		ValidationMode: serializer.ArrayValidationModeNone,
+		ValidationMode: serializer.ArrayValidationModeNoDuplicates,
+		UniquenessSliceFunc: func(next []byte) []byte {
+			return next[:1] // Check answer value for uniqueness
```

Comment:
Is it guaranteed that `next` is at least `1` in size?

Reply:
From what I understand from the serializer code it is first checked if the bytes can be parsed into the object, before the same bytes are used to pass to the UniquenessSliceFunc.